HW03/tim_bulat/spec/tim_bulat_hw_03_t_04_rspec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative '../tim_bulat_hw_03_t_04'
+
+RSpec.describe Homework3 do
+  describe '#task4' do
```

Comment:
you can create 1 subject here :  `subject { described_class.new.task4(string) }` and change string with the help of `let` in each context